src/View/Widget/DateTimeWidget.php

```diff
@@ -189,7 +189,7 @@ protected function formatDateTime($value, array $options): string
                 $dateTime = clone $value;
             } elseif (is_string($value) && !is_numeric($value)) {
                 $dateTime = new DateTime($value);
-            } elseif (is_int($value) || is_numeric($value)) {
+            } elseif (is_numeric($value)) {
```

Comment:
This should really only accept integer strings, but whatever.

Reply:
I agree, but it's not worth rocking the boat in 4.x, we can change it in 5.x.